operators/percona-xtradb-cluster-operator/1.15.0/manifests/percona-xtradb-cluster-operator.v1.15.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,750 @@
gsU0Tz5z3yoNTc9JcmqHrJbgqnprt+tOzUE9yqVZ9rwjD/mQtXVyhpNhtYiu5Pyj+Z/hWRDaEAAYk8E+xgh9JaN2XyKx2en6tGkn26qx/ctqIxTEMDqq+nerq7Zlsj7cAAAAAAAHzXf93/ZNqr0eFOrNR/Q3jD/S0VNdPRqmG+01301mm52x9+t4D4dwABePRL6Lp+0r/MZZab3bH7z8XPdHkmR3VQBo78up1XmU1i/zR59a6yi3Pbprn0tqOPXHb84+fmsdJqopjoV8uqUdM6tWQAGHsPBvrjup4qpUWGG2EXvx5sv9s26cxeux3R/M/wAK3V6qMdCj6ykBoVW41KipRcpNKKTbb2JJLFth7ETM4h856UXw7+tdWvt1ZSwpp8KUdkP2295VXK+nVMt3pNP/AKezTb6459/W1RzSQAB77gu/7WtVCjwqVYKX6E8Z/wClM+6KelVEOGpu+hs1XOyPv1PpQtmBAAAAAAAAKg6Zbp8ntFK0xX3ascub5VIbY49sW/dZB1VGJipqNjv9K3Vanq4x3T/fursiL0AAXj0S+i6ftK/zGWWm92x+8/Fz3R5Jkd1UAAPDbbqp2za1hL1lsffzIOp2+zf41Rie2Ei1qrlvhHLslq6mjkl5tRNdcWvgyqr2OvPq1x9Y/abTuNPXT92IaOSe+pFdkW/izynY7mfWrj6R+ydxp6qZbOxXRTsrxw1pc5bcOxbkWmm2yxYnOMz2yiXdXcucOUfJsCwRQCseljStQi7FRacpf+zJfljvVNdb3vku0h6m7/hH1aHZtDmfT1//AD+VVEJpAAAAsToaunyi0VbTJfdpRy4PnUntlh2RS95EvS0ZmalFvl/o26bUdfGe6P79lvk5lwAAAAAAADT6WXKtILJUoPBSaxpt/lqR2xfj8T4uUdOnCVo9TOnvRc8e587VKboylGScZRk4yT3xkng0+xplVMYbqJiYzHKXE8egF49Evoun7Sv8xllpvdsfvPxc90eSZHdVOi12ynYoqVScYRcoxTk8FrSeEVj1s8mYjm+6LdVc4pjLvPXwAAAAABXenPSJCxKVCxyU6zxU6q2wpfp4Sn+yIt7URHCnmvdv2mquYuXoxT2dc/iFQyk5tttttttt4tt7W2+LIDTxGOEMB6AAOVOm60oxinKUpKMUt8pN4JLtbR7EZeTMRGZ5Q+idE7lWj9kp0Fg5JY1GvzVJbZPx+Ba26OhThhdZqZ1F6bnh3NwfaKAAAAAAAAAKo6W9F3Tl5bSj914K0JcHujU7OD7iFqbX+cNLsutzHoK5/wCv4/CsiG0IBeXRL6Lp+0r/ADGWWm92x29fFz3R5Jkd1UhvS16Lqe0ofMRw1Pu1rs3xcd0+StNG9OrXcCUFJVaSwSp1G3qrlGW+PZtRDt36qODQarbLGonpTGJ7Y/mFg3V0qWS1JZ0alCXHFZsMeqcfqkSqdVRPPgo72yX6PYmKo8J8G/o6ZWCvutVHvlg/3OsXaJ60Krb9TTzokr6ZWChvtVHulrPwQm7RHWU7fqauVEo/evSpZLKnkxqV5cMFlQ75S2+CZyq1VEcuKdZ2O/X7cxTHjPgr7STTm13+nCUlSovFOnTxSkuU5b5dmxEW5fqr4LvS7ZY089KIzPbP8IzuOKxAAAABZvRJou6kvLasfurFWdPi90qnZwXeTNNa/wA5+jPb1rcR6Cif+34/K1yazQAAAAAAAAAAcKtJVouMknGSakntTT2NNDm9iZpnMKK080PlozU1oYys028t73Tf+XJ/B8e0rb1noTmOTZbduEamnFXtRz+fz/KJnBZry6JfRdP2lf5jLLTe7Y7evi57o8kyO6qQ3pa9F1PaUPmI4an3a12b4uO6fJRpWtiAAAAAAAAAAEr0D0PlpNU1p4xs0Gsx7nUf+XF/F8O072bPTnM8lZuO4RpqcU+1PL5fP8L2pUlRioxSUYpKKWxJLYkkWUcGNmZqnMuYeAAAAAAAAAAAA6bZZYW2nKnUipwkmpRaxTR5MRMYl90V1UVRVTOJhSmm+glTR9urS1qlm2vHfKiuU+cf6vHmV96xNHGOTW7fulGoxRXwq8+78J/0Sei6ftK/zGStN7tSb18XPdHkmR3VSG9LXoup7Sh8xHDU+7WuzfFx3T5KNK1sQAAAAAAAABM9CNBKmkDVWrrU7NseO6VZcoco/wBXhzJFmxNfGeSp3DdKNPmijjV5d/4XXY7LCw0406cVCEUlGKWCSLCIiIxDJV11V1TVVOZl3Hr4AAAAAAAAAAAAAAYaxA89gsFO7oOFKChDWlLVWxJyeLwXDaeRTEcIdLl2u5PSrnMvSeuaG9LXoup7Sh8xHDU+7WuzfFx3T5KNK1sQAAAAAAHtui6a18z1LPTlUlsxw82PXKW6J9U0TVOIcb1+3Zp6VycLV0U6MqVgaqWtxrVNjVNL8KD68fPfbguom29NEcauLN6zea7nq2fVjt6/0sFLAlKNkAAAAAAAAAAAAAAAAAAANPpZcX/kdllQ18vWcJKWrr4OMsVsxWO7mc7lHTpwlaPU/wCmuxcxlVV49F9tsv8AhulWXVJ037sv5IdWlrjk0lre9PV7WY+6P2rRe22Tz7LX/tg6i8YYnKbVcdSdRrtNXyrjxx5tfUsdSl51Kqu2nNfFHx0Z7HeLlE8qo8YKdjqVfNpVX2U5v4IdGewm5RHOqPGGwsui9ttfmWWv3wdNeM8D7i1XPU4V63TUc648c+SQXd0X221f4jpUV/VJzfux/k606aueaDd3vT0+zmUwubosstiwlXlO0S5P8On7ieL72zvRpaY58VXf3u9Xwtx0Y8Z8f0m9kssLHBQpwjCC3RilFLuRIiIjhCnrrqrnpVTmXcevkAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB//Z
```

Comment:
Could you please update the picture for operator? (We should use the same for all operators. Could you please check psmdb)
Sorry, I forgot to say :(